app/models/location_tag.rb

```diff
@@ -0,0 +1,4 @@
+class LocationTag < ActiveRecord::Base
+  attr_accessible :lat, :location, :long, :uid, :country, :state, :city
```

Comment:
`lon` or `lng` (I prefer `lon`) is a bit more standard than `long` -- would you mind changing this parameter name?